src/math/p5.Vector.js

```diff
@@ -524,7 +524,8 @@ p5.Vector.prototype.sub = function sub(x, y, z) {
 };
 
 /**
- * Multiply the vector by a scalar. The static version of this method
+ * Multiplies the vector by a scalar, multiplies the x, y, and z components from a vector, or multiplies
```

Comment:
Maybe reword this a bit to make the difference between multiplying by a scalar and multiplying by a vector.

Reply:
Sounds good, I'll see what I can do

Reply:
@limzykenneth also made some changes to the language here. Let me know if this makes more sense